cockroachdb/README.md

```diff
@@ -310,6 +310,7 @@ For details see the [`values.yaml`](values.yaml) file.
 | `image.repository`                                        | Container image name                                            | `cockroachdb/cockroach`                          |
 | `image.tag`                                               | Container image tag                                             | `v20.2.8`                                        |
 | `image.pullPolicy`                                        | Container pull policy                                           | `IfNotPresent`                                   |
+| `image.imagePullSecrets`                                  | Specify existing Image pull secret                              | `""`                                             |
```

Comment:
It's not clear to me how to use this from this description. Do we specify a K8s secret name? 

If so, it may help to include a hyperlink to some Kubernetes documentation on this feature, so others can know what that custom secret needs to contain.